topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md

```diff
@@ -112,7 +112,7 @@ We first perform an FBA (with COBRApy) optimizing the biomass reaction and recor
 
 > <hands-on-title>Calculating the flux of a target using Flux Balance Analysis (FBA)</hands-on-title>
 >
-> 1. Run {% tool [Flux balance analysis](toolshed.g2.bx.psu.edu/repos/iuc/rpfba/rpfba/5.12.1) %} with the following parameters:
+> 1. Run {% tool [Flux balance analysis](toolshed.g2.bx.psu.edu/repos/tduigou/rptools_rpfba/rptools_rpfba/6.3.2+galaxy0) %} with the following parameters:
```

Comment:
Can you maybe add your changes back to the IUC tool? That way we don't break the tool linage and don't need to update all Galaxy servers.

I pull request to this repo would be great: https://github.com/galaxyproject/tools-iuc
I will help you to get the changes in if you like.